code/index.html

```diff
@@ -1,22 +1,109 @@
+
+        <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+        <nav>
+            <a href="#">Contact</a>
+            <a href="#">Destinations</a>
+            <a href="#">Booking</a>
+        </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+
+
+    <section class="info">
+        <!-- <div class="image"> -->
+        <img src="pics/fiji.jpg" alt="vaccination image">
+        <!-- </div> -->
+        <div class="text">
+            <h1>Find deals on hotels, flights <br>and package trips.</h1>
+            <h3><i>We have destinations all over the world.</i></h3>
+            <button class="btn">Book here</button>
+        </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="destinations">
+        <article class="destination-box">
+            <h2>Greece</h2>
+            <img src="pics/greece.jfif" alt="Greece">
+        </article>
+        <article class="destination-box">
+            <h2>Spain</h2>
+            <img src="pics/spain.jfif" alt="Spain">
+        </article>
+        <article class="destination-box">
+            <h2>Italy</h2>
+            <img src="pics/italy.jpg" alt="wheather image">
+
+        </article>
+        <article class="destination-box">
+            <h2>Carribean</h2>
+            <img src="./pics/carribean.jpg" alt="Carribean">
+        </article>
+        <article class="destination-box">
+            <h2>South America</h2>
+            <img src="./pics/southamerica.jfif" alt="South America">
+        </article>
+        <article class="destination-box">
+            <h2>Africa</h2>
+            <img src="./pics/africa.jfif" alt="Africa">
+        </article>
+        <article class="destination-box">
+            <h2>Asia</h2>
+            <img src="./pics/asia.jfif" alt="Asia">
+        </article>
+        <article class="destination-box">
+            <h2>Cities</h2>
+            <img src="./pics/newyork.jpg" alt="New York">
+        </article>
+    </section>
+
+    <section class="staycation">
+        <div class="grid1">
+            <img src="./pics/westcoast.jpg" alt="Westcoast">
+        </div>
+        <div class="grid2">
+            <h2>Staycation</h2>
+            <h3>Experience our beautiful country.</h3>
+            <button class="btn">Book here</button>
+        </div>
+        <div class="grid3">
+            <img src="./pics/visby.jpg" alt="Visby">
+        </div>
+        <div class="grid4">
+            <img src="./pics/skiing.jpg" alt="skiing">
+        </div>
+        <div class="grid5">
+            <img src="./pics/stockholm.jpg" alt="Stockholm">
+        </div>
+        <div class="grid6">
+            <img src="./pics/sanddunes.jpg" alt="Wetscoast">
+        </div>
     </section>
-  </body>
-</html>
+
+
+    <footer>
+        <p>Created by Sara Stjernberg <span>@Technigo</span> August -20</p>
```

Comment:
Nice with the `<span>`-tag here 🤩 For the comment I left higher up regarding the `<i>`-tag, using a span and giving it a class of "italic" and styling that class in the CSS would be perfect. Well done on this👏